CHANGELOG_iOS.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [1.66.123](https://github.com/brave/brave-browser/releases/tag/v1.66.123)
+
+### Web3
+
+- Removed “Brave Fee” for all “Swap” transactions in Brave Wallet. ([#38566](https://github.com/brave/brave-browser/issues/38566))
```

Comment:
nit: the spacing is off with the `-`. Just remove the categories as you need to edit the PR either way.

Reply:
Ok, will do